cosmic_ray/worker.py

```diff
@@ -28,18 +27,20 @@
 from .parsing import get_ast
 from .testing.test_runner import TestOutcome
 from .work_item import WorkItem, WorkItemJsonDecoder
+import cosmic_ray.compat.json
 
 log = logging.getLogger()
 
 
 class WorkerOutcome:
```

Comment:
We could/should be using Enum here. It's part of the standard library from Python 3.4 onwards.